packages/firefly/src/AppRouter.tsx

```diff
@@ -145,14 +172,18 @@ export function BootstrappingRoute(props: BootstrappingRouteProps) {
     );
 }
 
+export type RenderRouterProviderFunction = (routes: Route[], providerProps: Omit<RouterProviderProps, "router">) => ReactElement;
```

Comment:
usually ReactNode is better than ReactElement in render function. 

it's more permissive, like if someone wants to pass a div as the root element

Reply:
Fixed!